Mathlib/Data/Sign/Defs.lean

```diff
@@ -209,13 +209,8 @@ def cast : SignType → α
   | pos => 1
   | neg => -1
 
-/-- This is a `CoeTail` since the type on the right (trivially) determines the type on the left.
-
-`outParam`-wise it could be a `Coe`, but we don't want to try applying this instance for a
-coercion to any `α`.
--/
-instance : CoeTail SignType α :=
-  ⟨cast⟩
+instance (s : SignType) : CoeDep SignType s α :=
```

Comment:
Since we want to get rid of the hom coercion instances, this will be the only remaining change once that all goes through.

This should get a docstring explaining why it is not a `CoeTail` or even `Coe`.